src/components/InactiveProjects.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react'
+import Grid from '@material-ui/core/Grid'
+
+import InactiveProjectListManager from './InactiveProjectListManager';
+import HomeInactiveProjects from './HomeInactiveProjects';
+import ClientInactiveProjectLists from './ClientInactiveProjectsList';
+
+const InactiveProjects = ({
+    history,
+    home,
+    clientId
+}) => {
+
+    const [showInactiveProjects, setShowInactiveProjects] = useState(false)
+
+    return (
+        <>
+            <InactiveProjectListManager
+                clientId={clientId}
+                showInactiveProjects={showInactiveProjects}
+                setShowInactiveProjects={setShowInactiveProjects}
+            />
+            {showInactiveProjects
+                ? (home
+                    ? (
+                        <Grid container>
+                            <HomeInactiveProjects
+                                history={history}
+                            />
+                        </Grid>
+                    ) : (
+                        <Grid container>
+                            <ClientInactiveProjectLists
+                                clientId={clientId}
+                                history={history}
+                            />
+                        </Grid>
+                    )
+                ) : null
```

Comment:
https://es.reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator

Reply:
This is a nested If. If I try to do the inline logical operator It won't work.

Reply:
```
{showInactiveProjects && (
    home
        ? (
            <Grid container>
                <HomeInactiveProjects
                    history={history}
                />
            </Grid>
        ) : (
            <Grid container>
                <ClientInactiveProjectLists
                    clientId={clientId}
                    history={history}
                />
            </Grid>
        )
)}
```